packages/ihe-gateway-sdk/src/models/patient-discovery.ts

```diff
@@ -45,11 +45,11 @@ export type PatientDiscoveryRequestIncoming = BaseRequest & {
 
 export type PatientDiscoveryResponseOutgoing =
   | (BaseResponse & {
-      patientMatch: boolean;
+      patientMatch: boolean | null;
```

Comment:
Can you refer to where we decided to return `null` for `patientMatch`, please? I'm wondering in which situations it makes sense, as by default I'd think it's either true or false.

Reply:
https://www.notion.so/XCPD-ITI-55-Interface-Incoming-0acba5d456eb4a1cb894d6b2d070de67?d=070e020c9dfe4959af5eda9c75942ce2&pvs=4#bf51482aaaa140628c1dc7098b388ecc